fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/WildcardExceptionMapper.java

```diff
@@ -56,7 +56,7 @@ public Response toResponse(final Exception e) {
                     .toResponse((RepositoryException)e.getCause());
         }
 
-        LOGGER.info("Exception intercepted by WildcardExceptionMapper: \n", e);
+        LOGGER.error("Exception intercepted by WildcardExceptionMapper: \n", e);
```

Comment:
Although you did not create this issue, please never dump an exception to the log. Change this to: `e.getMessage()`